### ISSUE TITLE
[Bug]: Error with /find command (mongobot)

### ISSUE THREAD

**Issue by CaptainRexPL** [Pilot1782/bad_copenheimer#95]
### What happened?

When I try to run /find [player] I get an error.

### Operating Sys

Windows

### Branch

- [X] Tried `dev-builds`

### Relevant log output

```shell
Task exception was never retrieved
future: <Task finished name='Task-40' coro=<find() done, defined at C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py:930> exception=TypeError("find() got an unexpected keyword argument 'player'")>
Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 970, in wrapper
    raise e
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 939, in wrapper
    return await coro(ctx, *args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by Pilot1782** [comment]
Lol ya, player doesn't work if you don't have a populated cluster, you can either add servers manually or run the scanner for months to find far less servers than you expected making you wonder if you should have used someone else's bot that actually works at scale and not some random persons project just figure out how Minecraft works with python only to realize that making something at scale is rather hard.

TLDR, populate your dataset or don't use player, cus it's might be broken atm, I have not tested it.

**Comment by CaptainRexPL** [comment]
Ok, I'll try to add some random servers then. But how to add a server to the DB.  There is a command called /add on the command list, but I can't find any implementation of this command.

**Comment by Pilot1782** [comment]
I just made a fix in the dev-builds branch that will allow you to add a server to your database if it's not in there by providing a host ip.

**Comment by CaptainRexPL** [comment]
Restarting in 2 hours
Client not authorised for the MESSAGE_CONTENT intent.
The HTTP client has exhausted a per-route ratelimit. Locking route for 0.999 seconds.
 All nameservers failed to answer the query _minecraft._tcp.160.251.44.110. IN SRV: Server 192.168.1.1 UDP port 53 answe'NoneType' object is not subscriptable
None
Done
Server not in database, adding...
Unknown interaction

  Error 10062 | Unknown Interaction.:
  Unknown interaction.
  Severity 40.
{'host': '160.251.44.110:25565', 'lastOnline': 1671269148.317257, 'lastOnlinePlayers': 0, 'lastOnlineVersion': '1.17.1', 'lastOnlineDescription': 'A Minecraft Server', 'lastOnlinePing': 2892, 'lastOnlinePlayersList': [], 'lastOnlinePlayersMax': 20, 'lastOnlineVersionProtocol': '756', 'favicon': None, '_id': ObjectId('639d8b217d179e48f833bb85')}
Unknown interaction
Task exception was never retrieved
future: <Task finished name='Task-26' coro=<find() done, defined at C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py:930> exception=LibraryException('\n  Error 10062 | Unknown Interaction.:\n  Unknown interaction.\n  Severity 40.')>
Traceback (most recent call last):
  File "E:\bad_copenheimer-dev-builds\mongoBot.pyw", line 180, in find
    await ctx.send(f"{text}")
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\context.py", line 501, in send
    await self._client.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\api\http\interaction.py", line 227, in create_interaction_response
    return await self._req.request(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\api\http\request.py", line 172, in request
    raise LibraryException(
interactions.api.error.LibraryException:
  Error 10062 | Unknown Interaction.:
  Unknown interaction.
  Severity 40.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 970, in wrapper
    raise e
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 939, in wrapper
    return await coro(ctx, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "E:\bad_copenheimer-dev-builds\mongoBot.pyw", line 190, in find
    await ctx.send("Server not found")
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\context.py", line 501, in send
    await self._client.create_interaction_response(
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\api\http\interaction.py", line 227, in create_interaction_response
    return await self._req.request(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\api\http\request.py", line 172, in request
    raise LibraryException(
interactions.api.error.LibraryException:
  Error 10062 | Unknown Interaction.:
  Unknown interaction.
  Severity 40.

**Comment by Pilot1782** [comment]
In the discord dev portal enable message send intent, reads to your server and to be safe reset the token.

**Comment by CaptainRexPL** [comment]
I enabled it and I changed the token, but except for "Client not authorised for the MESSAGE_CONTENT intent." the error is still the same.

**Comment by Pilot1782** [comment]
A fix has been pushed to dev-builds, so just download the new bot's code and replace it

**Comment by CaptainRexPL** [comment]
Now the problem is that the /find (host) command only detects a server ip with the port and when I just get:
`find None 160.251.44.110:25565 None None
string indices must be integers, not 'str'
Server not found
Duplicates removed`

**Comment by CaptainRexPL** [comment]
> As of right now, ports are not supported and are assumed as 25565, but it should work in the format of `127.0.0.1`

Unfortunately when I use just the ip without the port and the server port is set to 25565 then the scanner doesn't detect that as a minecraft server.

**Comment by Pilot1782** [comment]
Ok, its fixed now, and should work for you, I checked the IP you were testing and got it working.
![Screenshot_20221219_025746](https://user-images.githubusercontent.com/71522520/208532386-ae46c003-358c-4307-a63d-c0e42fc23c78.png)

**Comment by CaptainRexPL** [comment]
Yes, it works now. Now the only thing that doesn't work is player finder. 
`Task exception was never retrieved
future: <Task finished name='Task-51' coro=<find() done, defined at C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py:930> exception=TypeError("find() got an unexpected keyword argument 'player'")>
Traceback (most recent call last):
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 970, in wrapper
    raise e
  File "C:\Users\User\AppData\Local\Programs\Python\Python311\Lib\site-packages\interactions\client\models\command.py", line 939, in wrapper
    return await coro(ctx, *args, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: find() got an unexpected keyword argument 'player'`

**Comment by Pilot1782** [comment]
Yep I'm still working on that one.

**Comment by Pilot1782** [comment]
Player is added right now and working in dev-builds
